2016/help.html

```diff
@@ -0,0 +1,155 @@
+  <h2>We want to give you a great big, heartfelt
+    <span class="hearts">
+      <i class="fa fa-heart" aria-hidden="true"></i>
+      <i class="fa fa-heart" aria-hidden="true"></i>
+      <i class="fa fa-heart" aria-hidden="true"></i>
+      <i class="fa fa-heart" aria-hidden="true"></i>
+      <i class="fa fa-heart" aria-hidden="true"></i>
+      ‘thank you.’
+    </span>
+  </h2>
+</div>
+
+<div class="section section--white">
+  <h1>Our fantastic funders</h1>
+  <p>We are profoundly grateful to:</p>
+  <ul class="funders">
+    <li><a href="http://www.omidyar.com/">Omidyar Network</a>, <a href="http://www.potterfoundation.com/">The David &amp; Elaine Potter Foundation</a>, and <a href="https://www.opensocietyfoundations.org/">Open Society Foundations</a> — for their support across everything we do.</li>
+    <li><a href="http://www.hewlett.org/">The William and Flora Hewlett Foundation</a>, <a href="https://www.ucl.ac.uk/">University College London</a> and <a href="https://www.microsoft.com/en-gb/about/charities-communities/charity-support/">Microsoft</a> — for funding our research.</li>
+    <li><a href="https://indigotrust.org.uk/">The Indigo Trust</a> — for aiding our work with partners in sub-Saharan Africa.</li>
+    <li>The <a href="https://www.digitalnewsinitiative.com/">Google Digital News Initiative</a> — for funding the launch of Alaveteli Professional.</li>
+    <li><a href="http://www.ned.org/">National Endowment for Democracy</a> — for funding a WriteInPublic project for the Middle East and North Africa region</li>
+    <li><a href="http://www.jrct.org.uk">The Joseph Rowntree Charitable Trust</a> — for supporting our efforts to engage with a younger audience.</li>
+    <li><a href="http://www.makingallvoicescount.org/">Making All Voices Count</a> — for supporting Alaveteli for Liberia.</li>
+    <li><a href="https://www.google.com/">Google Inc.</a> — for sponsoring TICTeC 2017.</li>
+  </ul>
+</div>
+
+<div class="section section--no-padding">
+  <img src="{{ site.baseurl }}/assets/img/hands-on-keyboard.jpg" alt="" style="width: 100%;">
+</div>
+
+<div class="section section--white">
+  <h1>Our loyal donors</h1>
+  <p>Your money goes towards all sorts of things, including:</p>
+  <ul class="help-list">
+    <li>Keeping our servers spinning</li>
+    <li>Responding to our users’ support mail</li>
+    <li>Improving our sites</li>
+    <li>Maintaining our modest offices</li>
+    <li>Helping staff meet up face-to-face when needed</li>
+  </ul>
+  <p>Help us continue to provide sites that make a difference, in the UK and around the world
+<a href="https://www.mysociety.org/donate/?utm_source=annual-review&utm_medium=share-button&utm_campaign=2014">Become a mySociety donor now.</a></p>
+</div>
+
+<div class="section section--help-volunteers">
+  <h1>Our diligent volunteers</h1>
+  <p>You steer the policy on WhatDoTheyKnow, moderate the content and make sure we stay on the right side of the law.</p>
```

Comment:
Actually, I think the second person doesn't work here. It works for donors, because everyone is potentially one, but then you come to this section and think, 'Eh? No I don't!'.
So change 'you' to 'they' across these bullets please.